app/controllers/notifications_controller.rb

```diff
@@ -5,7 +5,6 @@ class NotificationsController < ApplicationController
   def index
     return unless user_signed_in?
 
-    @notifications_index = true
```

Comment:
`git log --patch` suggests this went away in https://github.com/forem/forem/pull/11027 where it was previously used to build the cache key for an inlined style.

Looks like a relic slated for removal